vast/companionads.go

```diff
@@ -10,14 +10,7 @@ type CompanionAds struct {
 func (companionAds *CompanionAds) Validate() error {
 	errors := make([]error, 0)
 
-	if len(companionAds.Required) != 0 {
-		if err := companionAds.Required.Validate(); err != nil {
-			ve, ok := err.(ValidationError)
-			if ok {
-				errors = append(errors, ve.Errs...)
-			}
-		}
-	}
+	// No necessary to validate Required which for VAST 3.0 only.
```

Comment:
// No need to validate Required which for VAST 3.0 only.

Reply:
// No need to validate Required which is for VAST 3.0 only.